core/contracts/oracle/implementation/Governor.sol

```diff
@@ -38,59 +40,30 @@ contract Governor is MultiRole, Testable {
     FinderInterface private finder;
     Proposal[] public proposals;
 
-    /**
-     * @notice Emitted when a new proposal is created.
-     */
+    /****************************************
+     *                EVENTS                *
+     ****************************************/
+
+    // Emitted when a new proposal is created.
     event NewProposal(uint indexed id, Transaction[] transactions);
 
-    /**
-     * @notice Emitted when an existing proposal is executed.
-     */
+    // Emitted when an existing proposal is executed.
     event ProposalExecuted(uint indexed id, uint transactionIndex);
 
+    /**
+     * @notice Construct the Governor contract.
```

Comment:
Do we need @notice comments for constructors? Maybe we should just provide @dev comments as neccessary for constructor methods that do a lot of complex actions such as in `PricelessPositionManager`

I'm just wondering, I do think there is a strong point in favor of having a @notice above every single method (constructors included)--for consistency